uktrade/data-hub-frontend
# Add "Cannot find" page for D&B company match

## Test instructions

1. Go to the D&B company matching form at `companies/COMPANY_ID/match`
2. Search for a company
3. Click `I cannot find the company I am looking for` and then `I still cannot find the company`
4. You should be redirected to a page titled `Unable to find matching company record`

## Screenshots

![image](https://user-images.githubusercontent.com/4199239/73090694-bfc99400-3ed0-11ea-9969-ace455528a29.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
